Jenkinsfile

```diff
@@ -163,19 +163,18 @@ node('docker && imgtec') {  // Only run on internal slaves as build takes a lot
                 [$class: 'FileBinding', credentialsId: 'opkg-build-private-key', variable: 'PRIVATE_KEY'],
                 [$class: 'FileBinding', credentialsId: 'opkg-build-public-key', variable: 'PUBLIC_KEY'],
             ]){
-                // Attempt to build quickly and reliably
+                sh "cp ${env.PRIVATE_KEY} ${WORKSPACE}/key-build"
+                sh "cp ${env.PUBLIC_KEY} ${WORKSPACE}/key-build.pub"
+
                 try {
-                    sh "cp ${env.PRIVATE_KEY} ${WORKSPACE}/key-build"
-                    sh "cp ${env.PUBLIC_KEY} ${WORKSPACE}/key-build.pub"
-                    sh "make -j4 V=s ${params.ALL_PACKAGES ? 'IGNORE_ERRORS=m' : ''}"
-                } catch (hudson.AbortException err) {
-                    // TODO BUG JENKINS-28822
-                    if(err.getMessage().contains('script returned exit code 143')) {
-                        throw err
+                    if(params.ALL_PACKAGES) {
+                        sh "make -j1 V=s IGNORE_ERRORS=m"
```

Comment:
My only concern is that the build would take a very long time (http://jenkins.creatordev.io/job/CreatorDev/job/openwrt/job/PR-296/buildTimeTrend took more than 10 hours!).  If we end up in a situation where we need to make an urgent release, then we will need to wait this long before getting a release candidate.

Why not go with the same logic as before i.e. try to build j4, if it fails, then build j1?